frontend/docker/Dockerfile

```
@@ -2,8 +2,15 @@
 #
 # SPDX-License-Identifier: CC-BY-SA-4.0
 
+# BASE
+FROM docker.io/node:18-alpine@sha256:bc329c7332cffc30c2d4801e38df03cbfa8dcbae2a7a52a449db104794f168a3 AS base
+RUN apk add --no-cache tini
+ENTRYPOINT ["tini"]
```

The [official documentation](https://github.com/krallin/tini#using-tini) sets the entrypoint slightly different.

```suggestion
ENTRYPOINT ["tini", "--"]
```